src/pug/blog/future_developments.pug

```diff
@@ -0,0 +1,159 @@
+extends _blog.pug
+
+block append subtitle
+  | Future developments update
+
+block content
+  h2 Future developments update
+  +news_header('6 November 2025', 'report_30', '')
+
+  h3 State of funding
+  p Small recap: When we launched Input Labs in #[em 2022] the plan was to fund the project via Patreon, later in #[em 2023] we started 3D-printing some parts and selling them in our shop, then since late 2023 we have been selling complete DIY kits in the shop, and that became our main funding source.
+  p This current funding model pays the bills, but unfortunately it #[em does not provide much income] to fund new research and development.
+  p We expected this situation to be temporary while we grew a bit more, but #[em organic growth] (without marketing investment) has been slowing down, and current #[em shipping restrictions] to the United States only made it worse.
```

Comment:
We have invested significant amounts of money into marketing (Enrique, Maria, G-Ads, LI, etc.). What was discontinued was IL own videos and approaching others.